web-console/4.0-designs/storage/object-storage-dashboard/object-storage-dashboard.md

```diff
@@ -0,0 +1,91 @@
+# Object Storage Dashboard￼
+
+## Vision
```

Comment:
Should we add some details here about when it will appear in the Console? Will it only show if the OCS operator is installed?

Reply:
@yuvalgalanti 

Let's have the following text for what the Vision is:
The main goal of the Object Service Dashboard is to provide an at-a-glance view of the Multi-Cloud Gateway (MCG) configuration, that is deployed as part of OpenShift Container Storage (OCS).  This dashboard would provide health and status information as well as reporting on key performance indicators for analysis and decision making.